### DISCUSSION TITLE
CTk Theme Builder now available

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#1659] [Showcase]
It's finally published to GitHub.

![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/4b3aa4d7-e653-468e-8060-afce81aa6ea0)

You can now download CTk Theme Builder from my GitHub repo page. Please click [here](https://github.com/avalon60/ctk_theme_builder/releases/tag/v2.0.0). Download the  ctk_theme_builder-2.0.0.zip  file.

Installation instructions are available [here](https://github.com/avalon60/ctk_theme_builder/blob/develop/docs/installation.md).

Documentation is well under way and can be found [here](https://github.com/avalon60/ctk_theme_builder/blob/develop/docs/user_guide.md).

**Comment by avalon60** [comment]
There are a bunch of my themes included. Some of these are work in progress, so please be aware that they may be subject to change with subsequent releases.

**Reply by alfmelchor** [reply]
Great job! Been looking forward to this project being published!

**Reply by avalon60** [reply]
Documentation is a bit raw. Just post questions if you aren't sure. Once installed you should be able to launch using the ctk_theme_builder command located in the install directory. For Linux and Mac this is a link pointing to ctk_theme_builder.sh (which you can call as an alternative).  For windows it's ctk_theme_builder.bat (you should be able to omit the .bat when typing as a command). 

I don't  have a Mac, so I haven't been able to test it on one. Only tested on Linux and Windows.

**Comment by Akascape** [comment]
@avalon60 It's awesome. I still can't believe you put so much work in there. Great project, thanks for sharing it with us.

**Reply by avalon60** [reply]
Thanks @Akascape. Have you got it running?

**Comment by avalon60** [comment]
Documentation is now complete and accessible via the repository [wiki](https://github.com/avalon60/ctk_theme_builder/wiki) .
